Source/ViewableController.swift

```diff
@@ -53,6 +53,15 @@ class ViewableController: UIViewController {
         return view
     }()
 
+    lazy var imageLoadingIndicator: UIActivityIndicatorView = {
+        let activityView = UIActivityIndicatorView(activityIndicatorStyle: .whiteLarge)
+        activityView.center = self.view.center
+        activityView.startAnimating()
+        activityView.alpha = 0
```

Comment:
Alternately, you could use `hidesWhenStopped` and just start and stop the animations -  that will take care of messing around with the alpha for you.